cloudwatch/src/lambda_function.py

```diff
@@ -67,6 +67,10 @@ def _parse_cloudwatch_log(log, aws_logs_data, log_type):
     log['invoked_function_arn'] = aws_logs_data['invoked_function_arn']
     log['type'] = log_type
 
+    if 'data_to_enrich' in aws_logs_data:
```

Comment:
Once I saw your PR I thought of a better way to implement it:
Instead of log['x'] = aws_logs_data['y']
We can do:
```suggestion
 log.update(aws_logs_data)
```